src/common/lib/command_mode.c

```
@@ -854,6 +841,56 @@ static bool process_brightness_select(const hid_keyboard_report_t* keyboard_repo
 }
 #endif
 
+// --- Public Functions ---
+
+void command_mode_init(void) {
```

Cppcheck (2.19.0)

[style] The function 'command_mode_init' is never used.

(unusedFunction)